tests/integration/test_keeper_disks/test.py

```diff
@@ -163,12 +165,22 @@ def test_logs_with_disks(started_cluster):
             look_behind_lines=2000,
         )
 
+        def get_local_log_files():
+            local_log_files = get_local_logs(node_logs)
+            start_time = time.time()
+            while len(local_log_files) != 1:
+                assert (
+                    time.time() - start_time < 60
+                ), "local_log_files size is not equal to 1 after 60s"
+                time.sleep(1)
+                local_log_files = get_local_logs(node_logs)
+            return local_log_files
+
         # all but the latest log should be on S3
+        local_log_files = get_local_log_files()
```

Comment:
The name `get_local_log_files` is kind of confusing, because it explicitly waits until (timeout or) the number of log files is 1.
```suggestion
        def get_single_local_log_file():
            local_log_files = get_local_logs(node_logs)
            start_time = time.time()
            while len(local_log_files) != 1:
                assert (
                    time.time() - start_time < 60
                ), "local_log_files size is not equal to 1 after 60s"
                time.sleep(1)
                local_log_files = get_local_logs(node_logs)
            return local_log_files

        # all but the latest log should be on S3
        local_log_files = get_single_local_log_file()
```